package.json

```diff
@@ -22,7 +28,35 @@
   },
   "dependencies": {
     "chokidar": "^3.5.1",
+    "electron-squirrel-startup": "^1.0.0",
```

Comment:
This look like a dependency that has no updates since 2016, is this really necessary? I didn't get the purpose of it.